documentation/modules/ROOT/pages/_partials/minikube-setup.adoc

```diff
@@ -65,10 +65,9 @@ istio-pilot-86bb4fcbbd-ptz7n               2/2     Running     0          6h35m
 istio-pilot-86bb4fcbbd-s9qxl               2/2     Running     0          6h34m
 istio-pilot-86bb4fcbbd-xd887               2/2     Running     0          6h34m
 istio-policy-5c4d9ff96b-tzkbw              2/2     Running     0          6h35m
+istio-security-post-install-xdxc7          0/1     Completed   0          6h35m
 istio-sidecar-injector-6977b5cf5b-9wsjw    1/1     Running     0          6h35m
-istio-statsd-prom-bridge-b44b96d7b-x85xr   1/1     Running     0          6h35m
 istio-telemetry-7676df547f-55p6c           2/2     Running     0          6h35m
-knative-ingressgateway-75644679c7-5xwx6    1/1     Running     0          6h27m
```

Comment:
this has actually being renamed:

https://github.com/knative/docs/blob/15488da77fff08facd2084283f48ce2663ba2b4c/docs/install/Knative-with-Minikube.md#installing-knative